packages/client/src/components/setList/index.js

```diff
@@ -48,18 +48,21 @@ function SetList({ mode }) {
 };
 
 function SetType({ allSets }) {
-    let expansionSets = allSets.map(({ name, set_type, code }, index) => {
+    // eslint-disable-next-line
+    let expansionSets = allSets.map(({ name, set_type, code, icon_svg_uri  }, index) => {
         if (set_type === 'expansion') {
-            return <Link to={`/set/${code}`} className='my-2' key={index}>{name}</Link>
+            return <Link to={`/set/${code}`} className='my-2' key={index}>
+                <img src={`${icon_svg_uri}`} className={'icon'}/> {name}
+            </Link>
```

Comment:
adding image to these sets, didn't want to do all of them, just showing its possible